src/components/App/App.tsx

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react'
-import { default as DappApp } from 'decentraland-dapps/dist/containers/App'
 
+import { Navbar, Page, Footer, Locale } from 'decentraland-ui'
 import * as languages from 'modules/translation/languages'
+import HomePageHero from 'components/HomePageHero'
 import { Props } from './App.types'
 
 export default class App extends React.PureComponent<Props> {
   render() {
-    const { children } = this.props
+    const { isHomePage, children } = this.props
+
     return (
-      <DappApp activePage="builder" locales={Object.keys(languages)} onSignIn={undefined}>
-        {children}
-      </DappApp>
+      <>
```

Comment:
Should we add `isFullscreen` and `isOverlay` to `<App>` and use that one instead?